packages/e2e-tests/specs/block-deletion.test.js

```diff
@@ -30,7 +30,7 @@ describe( 'block deletion -', () => {
 	beforeEach( addThreeParagraphsToNewPost );
 
 	describe( 'deleting the third block using the Remove Block menu item', () => {
-		it( 'results in two remaining blocks and positions the caret at the end of the second block', async () => {
+		it.skip( 'results in two remaining blocks and positions the caret at the end of the second block', async () => {
```

Comment:
The one I'm hesitant because in my experience at least it has shown to be unreliable only when using "popular plugins" and the fact that the popular plugins test didn't prove to be succesful IMO, I wonder if we should remove those instead.